src/schema.toml

```diff
@@ -36,6 +36,7 @@ features.require_account_for_join = { category = "rooms", type = "boolean", name
 features.default_room_size = { category = "rooms", type = "number", name = "Default room size", description = "Default room size for new rooms. This does not include users in the lobby." }
 features.max_room_size = { category = "rooms", type = "number", name = "Maximum room size", description = "Maximum room size visitors can set." }
 
+features.show_feature_panels = { category = "features", type = "boolean", internal = "true" }
```

Comment:
Thanks for making this a toggle in the Admin panel! Was my first thought looking at the feature.